test/test-malformed-pointers.sh

```diff
@@ -83,7 +83,9 @@ begin_test "empty pointers"
   pushd .. >/dev/null
     clone_repo "$reponame" "$reponame-assert"
 
-    [ "0" -eq "$(grep -c "empty.dat" clone.log)" ]
+    # The below assertion is known to be broken on Git v2.14.next.
```

Comment:
@larsxschneider I'm not sure, but it appears that this is fixed on 'master', whatever was causing it, so I removed the assertion skip in 8bd7aac.